docs/it-manual/testing-and-qa/testing-resources.md

```diff
@@ -9,26 +9,9 @@ Product managers, user experience designers, and others may need to test out Civ
 * CiviForm admins
 * Program admins
 
-Logging in as applicants, CiviForm admins, or Program admins is pretty easy -- there should be buttons for those directly on [staging.seattle.civiform.com](https://staging.seattle.civiform.com):
+Logging in as applicants, CiviForm admins, Program admins, or TIs is pretty easy -- there should be buttons for those directly on [[staging.seattle.civiform.com](https://staging.seattle.civiform.com)](https://staging-aws.civiform.dev):
```

Comment:
nit, none blocking

It's good practice to either spell out acronyms the first time you use them on the current page (E.g. Trusted Intermediaries (TIs)) or to link to a resource that explains the acronym

Reply:
Yes! This PR collapsed it but it actually does explain `Trusted intermediaries (TIs)` a few lines above this.